csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/DeprecationTest.cs

```diff
@@ -0,0 +1,30 @@
+namespace DeprecationTests
+{
+    [TestClass]
+    public class DeprecationTest
+    {
+        [TestMethod]
+        public void GivenInputString_WhenReverseString_Deprecated_ThenReturnReversedString()
+        {
+            string input = "Hello, World!";
```

Comment:
Since this string is used in both tests, along with the same expected output string, we can move both of them to the class level and make them constants.